enzy_htp/__init__.py

```diff
@@ -38,16 +34,15 @@
     write_data,
 )
 from ._config import config
+
 from .preparation import PDBLine, PDBPrepper, read_pdb_lines
 from .chemical import ResidueType
+from ._config import Config
 
-# from ._interface import Interface
-# from .mutation import mutate_pdb, restriction_object, MutationRestrictions
-# from .geometry import sample_geometries
+config = Config()
```

Comment:
I think we discussed about instantiate the Config() in the init file of _config as line 36 shows

Reply:
Done!